### ISSUE TITLE
[craftedv2beta] `crafted-updates` error when the branch is not master

### ISSUE THREAD

**Issue by bacchist** [SystemCrafters/crafted-emacs#309]
Starting a new issue to address this on its own, as requested. This is concerning the message being displayed on the Crafted Emacs startup page that reads "Crafted Emacs status could not be determined.

> Regarding the second, could you open a second issue, turn on debug-on-error and then provide any stack trace you might get when you see that error? That would be very helpful!

It did not enter the debugger, but according to the help, debug-on-error:
> Does not apply to errors handled by ‘condition-case’ or those matched by ‘debug-ignored-errors’.

It looks like this is the case (lines 165-167 of crafted-startup-config.el):
```
           (format "%s\n" (condition-case nil
                              (crafted-updates-status-message)
                            (error "Crafted Emacs status could not be determined.")))))
```

I followed thread a bit and got to `crafted-updates--get-new-commit-count`, which runs `git rev-list` with a path option of `master..origin/master`

Changing this to `craftedv2beta..origin/craftedv2beta` appears to be a temporary solution, but I'm going to post this issue anyway so a better solution can be considered.

**Comment by jeffbowman** [comment]
Ah!  Good stuff, saw your PR before seeing this.  I should move the comment from the PR here for discussion.

**Comment by jeffbowman** [comment]
> I wonder if there is a way to make this more general rather than depending on strings - can we detect the branch and assume (or maybe introduce a variable) the cloned repo name is "origin"? Then the "craftedv2beta..origin/craftedv2beta" string would be correct regardless of the branch the user is on?

Pulled from the related PR (See #310).

**Comment by jeffbowman** [comment]
Fixed in #310, closing here.
